src/Cake.Tfs/PullRequest/TfsPullRequest.cs

```diff
@@ -196,6 +198,26 @@ public int CodeReviewId
             }
         }
 
+        /// <summary>
+        /// Gets the name of the source branch
+        /// </summary>
+        /// Returns <see cref="string.Empty"/> if no pull request could be found and
+        /// <see cref="TfsPullRequestSettings.ThrowExceptionIfPullRequestCouldNotBeFound"/> is set to <c>false</c>.
+        /// <exception cref="TfsPullRequestNotFoundException">If pull request could not be found and
+        /// <see cref="TfsPullRequestSettings.ThrowExceptionIfPullRequestCouldNotBeFound"/> is set to <c>true</c>.</exception>
+        public string SourceRefName
+        {
+            get
+            {
+                if (!this.ValidatePullRequest())
+                {
+                    return string.Empty;
+                }
+
+                return this.pullRequest.SourceRefName;
+            }
+        }
```

Comment:
This is adds a separate enhancement. To have this appear in the release notes properly I've created https://github.com/cake-contrib/Cake.Tfs/issues/36

Reply:
@pascalberger Do you mean this should be pulled out from this branch and merged as a separate PR?

Reply:
A separate PR would be a cleaner approach. If you want you can move it to a separate PR, but I'm OK to keep it here. The important thing is that an issue exists for it so that it appears in the release notes later.

Reply:
@pascalberger Makes sense. I've pulled out that part. As soon as this PR is merged, I'll issue a new one with just that change, referencing the new issue you created.